tests/test_game.py

```diff
@@ -0,0 +1,7 @@
+import unittest
+from server.game import Game
+
+
+class TestServer(unittest.TestCase):
+    def setUp(self):
+        self.game = Game('player1', 'player2')
```

Comment:
What do you want to test with this?

Reply:
The coverage does not allow to upload code without test